react/components/notes.css

```diff
@@ -0,0 +1,177 @@
+.notesContainer {
+  width: 100%;
+  min-width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.addNotesIcons {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
+
+.notesButtonContainer {
+  display: flex;
+  justify-content: flex-end;
+  gap: 1rem;
+  padding-top: 1rem;
+}
+
+.notesSubmitButtonContainer {
```

Comment:
remove empty css classes

Reply:
Can't remove this. If removed the block class will be undefined.